### DISCUSSION TITLE
关于内存优化

### DISCUSSION THREAD

**Discussion by SadEPa0628** [Meloong-Git/PCL#2566] [讨论]
既然PCL拥有优化内存如此牛X的功能干嘛不弄成自动的，内存不足就自动优化。

**Comment by Yue-cn** [comment]
有机械装系统的...而且这只是将在(物理**内存**)中的数据转运到(虚拟**内存**)

**Comment by SuoHouDaiShi** [comment]
首先，你要考虑部分仍然使用**旧设备的玩家们**，他们的设备**一般不会有固态硬盘**，而如果每次开启启动器自动执行优化，那不好意思……准备迎接**卡顿**吧。
其次你得考虑我这种**有固态硬盘**<del>但是傻逼</del>的人群，具体表现为**系统在SSD中，游戏在机械硬盘/移动机械硬盘，这样一来会对SSD造成魔法伤害，不可规避的卡顿。**
最后，**如果你是刚上网的**，建议去读一读《提问的智慧》这本书，能**帮到你大忙。**
